lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -6,7 +6,13 @@ class Processor::MetricsFileInfo < Processor
       # @type self: SchemaClass
       let :issue, object(
         lines_of_code: integer?,
-        last_committed_at: string
+        last_committed_at: string,
+        number_of_churn_commits: integer,
```

Comment:
[suggestion] I feel a bit redundant about `churn`. How about this? ⬇️ 

```suggestion
        number_of_commits: integer,
```

Reply:
Fixed. commit 93732f7a3a93ab53c0ea9502bc80de21768270e2